adapter/internal/adapter/adapter.go

```diff
@@ -277,16 +287,19 @@ func fetchAPIsOnStartUp(conf *config.Config) {
 		if data.Resp != nil {
 			// For successfull fetches, data.Resp would return a byte slice with API project(s)
 			logger.LoggerMgw.Debug("Pushing data to router and enforcer")
+			health.SetControlPlaneRestAPIStatus(true)
 			err := synchronizer.PushAPIProjects(data.Resp, envs)
 			if err != nil {
 				logger.LoggerMgw.Errorf("Error occurred while pushing API data: %v ", err)
 			}
```

Comment:
I think it is better to consider this err when pushing API Projects
set health if there is no error when pushing projects

The error may occur when unzipping artifacts, Error reading deployment descriptor if API file not found defined in the descriptor. @Rajith90 @pubudu538 shall I consider this err as well, if there is an error, then make it unhealthy?